ewah/dag_factories/dag_factory_atomic.py

```diff
@@ -25,7 +25,7 @@ def dag_factory_atomic(
     target_schema_suffix: str = "_next",
     target_database_name: Optional[str] = None,
     default_args: Optional[dict] = None,
-    schedule_interval: Union[str, timedelta] = timedelta(days=1),
+    schedule_interval: Optional[Union[str, timedelta]] = None,
```

Comment:
Comment (blocking): I am not a fan of changing default behaviour. Is this needed, because it defaults to the timedelta even if None is supplied explicitly? I am not certain how Python behaves in this case

Reply:
Fair, I tested and when `None` is passed explicitly Python keeps it as `None`. 

I changed it back to have a default value as before.